manifests/check.pp

```diff
@@ -130,6 +138,9 @@
   Variant[Undef,Enum['absent'],Hash] $subdue = undef,
   Variant[Undef,Enum['absent'],Hash] $proxy_requests = undef,
   Variant[Undef,Enum['absent'],Hash] $hooks = undef,
+  Optional[Enum['influxdb_line', 'graphite_plaintext', 'nagios_perfdata',
```

Comment:
Should this be `String[1]`? Could you have custom output formats? @asachs01

Reply:
According to this: https://docs.sensu.io/sensu-enterprise/latest/built-in-mutators/ those values are the only ones allowed, I would think custom is not allowed as that would likely require custom code being a part of Sensu.

Reply:
Aaron let me know that you could write your own mutator, so let's keep it broad.

Reply:
I updated to just require a string.